CHANGELOG.md

```diff
@@ -9,6 +9,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Added timestamped conversation transcript support:
+
+  - New `TranscriptProcessor` that builds chronological transcripts.
+  - New frame types `OpenAILLMContextUserTimestampFrame` and
+    `OpenAILLMContextAssistantTimestampFrame`.
+  - Messages emitted with ISO 8601 timestamps indicating when they were spoken.
+  - Supports all LLM formats (OpenAI, Anthropic, Google) via standard message
+    format.
+  - New examples: `28a-transcription-processor-openai.py`,
+    `28a-transcription-processor-openai.py`, and
+    `28a-transcription-processor-openai.py`.
```

Comment:
Should be ...-anthropic.py and -gemini.py

Reply:
Fixed.